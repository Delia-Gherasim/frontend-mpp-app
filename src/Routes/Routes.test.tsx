import {createBrowserRouter} from 'react-router-dom';
import {describe, expect, test} from 'vitest';
describe('Router configuration snapshot', () => {
    test('Snapshot test', () => {
        expect(createBrowserRouter).toMatchSnapshot();
    });
});
