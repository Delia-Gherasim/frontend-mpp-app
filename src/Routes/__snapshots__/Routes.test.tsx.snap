// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Router configuration snapshot > Snapshot test 1`] = `[Function]`;
